src/main/java/com/work/management/service/employee/impl/EmployeeServiceImpl.java

```diff
@@ -113,5 +147,26 @@ public EmployeeDto updateEmployeeEntity(EmployeeDto employeeDto) {
         }).map(Optional::get)
         .collect(toImmutableList());
   }
+
+  @Override
+  public String createDocument(final EmployeeDocumentDto employeeDocumentDto) {
+
+    try {
+      Map employeeDtoMapper = objectMapper.convertValue(employeeDocumentDto, Map.class);
+      final IndexRequest indexRequest = new IndexRequest(
+          elasticConfigProperties.getIndex().getName(), "_doc",
```

Comment:
Limit code in try block, only put that line in try block which can throw expection.